### ISSUE TITLE
Fast Reload in React App

### ISSUE THREAD

**Issue by halilibrahimcelik** [wxt-dev/wxt#157]
### Feature Request
It would be super cool and development friendly to take instant snapshot of changes while creating the application components. As you can see, in order to see changes I had to close and open the extention again and again.  I searched for possible config solutions, but I could not see any outcome to solve this hot reload issue.

https://github.com/wxt-dev/wxt/assets/92088301/38963a8e-edf6-4c93-abf2-b66cb9fe2950

**Comment by aklinker1** [comment]
Hmm, this should already be working like you want. It looks like this is a problem with tailwind not finding the new class after you save the file, at least until you re-open the popup.

> "No utility classes were detected in your source files. If this is unexpected, double-check the `content` option in your Tailwind CSS configuration"

It could still be a bug in WXT or something misconfigured locally. Can you share this project with me, and I can look into this? You can just delete the `node_modules` folder, zip up your project, and upload it in a comment here.

**Comment by aklinker1** [comment]
OK, I can reproduce with the react template. Let me look into fixing that and get back to you.

**Comment by aklinker1** [comment]
Looks like WXT isn't adding the react-refresh runtime for some reason.

```html
<script type="module">
import RefreshRuntime from "/@react-refresh"
RefreshRuntime.injectIntoGlobalHook(window)
window.$RefreshReg$ = () => {}
window.$RefreshSig$ = () => (type) => type
window.__vite_plugin_react_preamble_installed__ = true
</script>
```

This should appear in the HTML file's `head`, but it's missing in the popup.

This is added by vite's React plugin, so I'm not sure why it's not getting applied to the `popup.html`. I'm still looking into it, but I've got to leave for a bit. I'll try fixing this later tonight.

**Comment by aklinker1** [comment]
Alright, I have a working solution. I'll get it released tomorrow, there's still some cleanup work to do.

https://github.com/wxt-dev/wxt/assets/10101283/c210eb76-2de7-478a-9617-b6bb11f0434a

**Comment by aklinker1** [comment]
No problem! Released in [v0.7.4](https://github.com/wxt-dev/wxt/releases/tag/v0.7.4), feel free to re-open if it doesn't work for you.

**Comment by halilibrahimcelik** [comment]
It works thank you!!! 🙏🙏🙏
